roles/rhods_test_jupyterlab/tasks/main.yml

```
@@ -176,6 +176,39 @@
     oc delete ev -n {{ rhods_test_namespace }} --all > /dev/null
   failed_when: false
 
+- name: Conditionally update the default notebook size
+  environment:
+    KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+  when: rhods_test_jupyterlab_notebook_cpu | length > 0
+  block:
+    - name: Retrieve the configmap containing the default notebook size
+      environment:
+        KUBECONFIG: '{{ sut_cluster_kubeconfig }}'
+      command:
+        oc get configmap -o json --namespace redhat-ods-applications odh-jupyterhub-global-profile 
```

can you update this (for consistency with the rest of the code base)
```suggestion
        oc get configmap -ojson -n redhat-ods-applications odh-jupyterhub-global-profile 
```
(usually I consider better to use the long flags, but `oc ... -n` is so common that I use the short version for clarity)